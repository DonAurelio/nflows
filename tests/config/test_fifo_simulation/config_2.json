{
    "dag_file": "./tests/workflows/test_fifo_simulation/config_2.dot",
    "scheduler_type": "fifo",
    "scheduler_params": [
        "fifo_prioritize_by_core_id=yes",
        "fifo_prioritize_by_exec_order=yes"
    ],
    "mapper_type": "simulation",
    "mapper_mem_policy_type": "default",

    "core_avail_mask": "0x0F",
    "flops_per_cycle": 1000000,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1,

    "distance_matrices": {
        "latency": "./tests/system/test_fifo_simulation/2_lat.txt",
        "bandwidth": "./tests/system/test_fifo_simulation/2_bw.txt"
    },

    "log_base_name": "./tests/output/test_fifo_simulation/config_2",
    "log_date_format": ""
}
