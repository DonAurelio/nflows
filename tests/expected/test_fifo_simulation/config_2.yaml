runtime:
  core_availability:
    0: {avail_until: 70}
    1: {avail_until: 50}
    2: {avail_until: 40}
    3: {avail_until: 70}

trace:
  exec_name_total_offsets:
    Task_4: {start: 50, end: 70, payload: 10}
    Task_3: {start: 50, end: 70, payload: 10}
    Task_5: {start: 20, end: 40, payload: 10}
    Task_2: {start: 20, end: 50, payload: 10}
    Task_1: {start: 0, end: 20, payload: 10}
